crypto/stack/stack.c

```diff
@@ -191,7 +191,7 @@ static int internal_find(OPENSSL_STACK *st, const void *data,
                         ret_val_options);
     if (r == NULL)
         return (-1);
-    return (int)((char **)r - st->data);
+    return (int)((const char **)r - st->data);
```

Comment:
Not sure what you’re saying.  We need to cast to be able to subtract pointers.  And I think the return value is proper.